tools/packaging/static-build/kernel/Dockerfile

```diff
@@ -5,6 +5,10 @@
 FROM ubuntu:20.04
 ENV DEBIAN_FRONTEND=noninteractive
 
+ARG http_proxy
+ARG https_proxy
+ENV http_proxy ${http_proxy}
+ENV https_proxy ${https_proxy}
```

Comment:
Isn't `$ARG` enough?

Reply:
If no proxy is added, network problems will occur frequently in the intranet environment.

Reply:
No, I meant: You make Docker aware of `http_proxy` through `ARG http_proxy`, so is `ENV http_proxy ${http_proxy}` necessary?

Reply:
ARG exists at build time and can be used as a variable in the Dockerfile. ENV is an environment variable after the container is built and cannot be used as a parameter in the Dockerfile. I want to use http_proxy as the environment variable of the container. For convenience, both ARG and ENV are required.